packages/text_extraction/Dockerfile

```diff
@@ -0,0 +1,64 @@
+# Build the Spring Boot application
+FROM python:3.11-slim-bookworm AS text_extraction_taskrunner_builder
+
+###### DEV ONLY ######
+#VVVVVVVVVVVVVVVVVVVVV
+
+# These deps are installed only for use during local development
+
+# Install OpenJDK
+RUN apt-get update && apt-get install -y --no-install-recommends \
+	ocrmypdf \
+	openjdk-17-jdk-headless && \
+	rm -rf /var/lib/apt/lists/*
```

Comment:
Is this up to date with the latest drama with debian and openJDK?

Reply:
looks like it